src/main/java/com/susmitha/wordtrace/WordTraceApp.java

```diff
@@ -21,11 +23,22 @@ public static void main(String[] args) {
         }
         String inputFilePath = args[0];
         String searchWord = args[1];
-        System.out.println(WordTraceConstants.MESSAGE_INPUT_FILEPATH + inputFilePath);
-        System.out.println(WordTraceConstants.MESSAGE_SEARCH_WORD + searchWord);
-        System.out.println(WordTraceConstants.MESSAGE_FILEPATH_CORRECT);
-        System.out.println(WordTraceConstants.MESSAGE_PROCESSING);
-        int wordCount = FileProcessor.countWordOccurrences(inputFilePath, searchWord);
-        System.out.println(WordTraceConstants.MESSAGE_COUNT_WORD);
+        //Creating object for File class
+        File inputFile = new File(inputFilePath);
+        String inputFileName = inputFile.getName();
+        System.out.println(
+                WordTraceConstants.MESSAGE_INPUT_FILEPATH + inputFilePath + "\n" +
+                        WordTraceConstants.MESSAGE_SEARCH_WORD + searchWord + "\n" +
+                        WordTraceConstants.MESSAGE_FILEPATH_CORRECT + "\n" +
+                        WordTraceConstants.MESSAGE_PROCESSING
+        );
+        //Creating object for wordCounter class
+        WordOccurrenceCounter wordCounter = new WordOccurrenceCounter();
```

Comment:
Pls rename the variable
```suggestion
        WordOccurrenceCounter wordOccurenceCounter = new WordOccurrenceCounter();
```